.github/workflows/publish-and-release.yaml

```diff
@@ -0,0 +1,136 @@
+name: Release c2patool crate and publish binaries
+
+on:
+  push:
```

Comment:
I'm assuming that this file will replace the current publish.yml and have `workflow_dispatch` again?

Reply:
yes